lumigator/infra/mzai/helm/lumigator/values.yaml

```diff
@@ -20,8 +20,8 @@ serviceAccountName: ""
 
 summarizerWorkDir: ""
 
-mistralSecretName: mistral-key
-mistralSecretKey: MISTRAL_API_KEY
+mistralSecretName: ""
```

Comment:
Perhaps change Mistralsecretname to existingMistralSecret to make it more clear

Reply:
Ok